### DISCUSSION TITLE
Dropping data variables in `InferenceData`

### DISCUSSION THREAD

**Discussion by mortonjt** [arviz-devs/arviz#1905] [Q&A] [answered]
Just out of curiosity, it is possible to drop data variables from `InferenceData` objects?  For instance, one may want to drop all of the transformed variables from a stan run to save memory.  I see that there is a [isel](https://arviz-devs.github.io/arviz/api/generated/arviz.InferenceData.isel.html#arviz.InferenceData.isel) method, but supporting something like [xarray.Dataset.drop](http://xarray.pydata.org/en/v0.13.0/generated/xarray.Dataset.drop.html) would be more straightforward.

EDIT : I found that just deleting variables can resolve this, for instance
```python
del inf['posterior']['useless_variable']
```

**Comment by OriolAbril** [comment, marked as answer]
As a starting point you can do `inf["posterior"].drop()` as all groups are xarray datasets, but we could extend the drop method to work with `inf.drop()` directly so you can drop variables from both posterior and prior at the same time for example
